app/controllers/admin/members_controller.rb

```diff
@@ -35,4 +39,47 @@ def permitted_params
     params.require(:member).permit %i(email username approved first_name
     last_name phone postal_street postal_zip postal_city)
   end
+
+  def load_website
+    if current_user.admin?
+      @website = current_website
+    else
+      @website = current_user.websites.find(current_website.id)
```

Comment:
On second thought, yes it's safe and should remain as is. `current_website` is not one that is authorized to belong to the current_user, just what the current subdomain points to, so we have to check here that it belongs to the user.